portfolio/src/main/webapp/index.html

```diff
@@ -5,34 +5,67 @@
   <title>Fiza Goyal</title>
   <link rel="stylesheet" href="style.css">
   <script src="script.js"></script>
+
+  <link href="https://fonts.googleapis.com/css2?family=Slabo+27px&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
 </head>
 <body>
-  <div class="header" id="stickyHeader">
+  
+  <!--------------HEADER-------------->
+  <header>
     <h2 class = "title">Fiza Goyal</h2>
     
     <!-- The navigation menu -->
-    <div class="navbar">
-      <a class="active" href="#about">About</a>
-      <a href="#experience">Experience</a>
-      <a href="#">Projects</a>
-      <a href="#">Contact</a>
-  </div>
-  </div>
+    <nav class = "top-navbar">
+      <ul>
+        <a class="active" href="#about"><li>Home</li></a>
+        <a href="#about"><li>About</li></a>
+        <a href="#experience"><li>Experience</li></a>
+        <a href="#projects"><li>Projects</li></a>
+        <a href="#contact"><li>Contact</li></a>
+        <a href="/resources/FizaGoyalResume.pdf" target="_blank" class = "resume"><li>Resume</li></a>
+      </ul>
+    </nav>
+    <div style="clear: both"></div>
+  </header>
+  
+  
   <div id = "content">
-    <div id = "about">
-        <a name = "about"></a>
-        <h1>Hi! I'm Fiza</h1>
-        <p>I'm currently an incoming junior studying Computer Science at the University of Illinois at Urbana-Champaign.</p>
-        <img class = "circular" src="resources/headshot_square.png" alt="My headshot!" width="300" height="300">
-        <a href="/resources/FizaGoyalResume.pdf" target="_blank">Resume</a>
-        <span class="circle"></span>
-        <p>Click here to get a random message:</p>
-        <button onclick="addRandomGreeting()">Hello</button>
-        <div id="greeting-container"></div>
+  
+  <!--------------HOME-------------->
+  <section class = "bg-color-section home-banner">
+      <div id = "home" class = "inner-section">
+        <div class = "front-bio">
+            <h3>Hi!</h3>
+            <h1>I'm Fiza.</h1>
```

Comment:
It looks like here you are using the header tags not to create a logical outline structure, but rather as a way to style the text you want. You want to keep the header tags for creating the logical structure.

So in this example, the "Hi!" is logically part of the previous logical unit, and a new, level-1 section titled "I'm Fiza" begins after it. It also breaks the header-level structure to start with an <h3>.

The handling of font sizes and bolding should be done via CSS, which is what should be the primary source of visual presentation information.